qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/controllers/CreateCustomerController.groovy

```diff
@@ -46,10 +46,11 @@ class CreateCustomerController {
         }
 
         try {
-            Customer customer = new Customer(firstName, lastName, academicTitle, email, affiliations as List<Affiliation>)
+            Customer customer = new Customer.Builder(firstName,lastName,email).title(academicTitle).affiliations(affiliations).build()
```

Comment:
NIT: did you format this line also? I am missing spaces behind the commas for example.

Reply:
Your eyesight is amazing! Yes i rewrote the line during the move to the builder pattern.